### DISCUSSION TITLE
New release scheduling?

### DISCUSSION THREAD

**Discussion by dropwhile** [neutrinolabs/xorgxrdp#264] [General]
Is there a new release scheduled any time in the near future?  
Just curious, as it would be nice to get the scrolling fixes in a package in the distro I use (alpine).

**Comment by metalefty** [comment]
Usually, we make a release 3 times per year. However, there're very few changes in recent years so we will make on-demand release now.

There are no changes since we made a scroll speed mitigation release, so there's nothing to release right now and no plan to release.

**Reply by dropwhile** [reply]
I was under the impression that https://github.com/neutrinolabs/xorgxrdp/pull/234 didn't make it into the sept 9th release (`v0.9.19`), only the earlier `XRDP_XORG_TOUCHPAD_SCROLL_HACK`. 

Am I incorrect in that assumption?

**Reply by metalefty** [reply]
See release note: https://github.com/neutrinolabs/xorgxrdp/releases/tag/v0.9.19
